app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -450,6 +451,15 @@ struct DeviceDelegateOpenXR::State {
       sessionBeginInfo.primaryViewConfigurationType = viewConfigType;
       CHECK_XRCMD(xrBeginSession(session, &sessionBeginInfo));
       vrReady = true;
+
+      // If hand tracking is supported, we want to emulate a default interaction
+      // profile, so that if Wolvic is launched without controllers active, we can
+      // still use hand tracking for emulating the controllers.
+      // This is a temporary situation while we don't implement WebXR hand tracking
+      // APIs.
+      if (mHandTrackingSupported) {
+          UpdateInteractionProfile(OculusTouch.path);
```

Comment:
I've just realized that this is wrong. This basically breaks any other flavour that comes with handtracking support. You have to use the proper profile for each device. I'd suggest creating a new method with some ifdefs that returns the proper interaction profile depending on the build.

Reply:
Right, I totally forgot about Pico.

I added a new state function that returns a default profile path for current device, as suggested. Thanks!